tests/functional/tests.py

```diff
@@ -68,31 +71,87 @@ def read(self, amt=64*1024):
         self._offset_location += len(data)
         return data
 
+def initialize_log_output(meth, alert = None):
+    # Initialize and return log content in log_output dictionary
+    # Collect args in args_arr
+    args_list = inspect.getargspec(meth).args
+    # Remove the first args_list element, which is always "self"
+    del args_list[0]
+    # Initialize the args , (arg: value), dictionary
+    args_dict = {value: '' for key, value in enumerate(args_list)}
+    # Create and return log output content
+    return {'name': 'minio-py',\
+            'function': meth.__name__+'('+', '.join(args_list)+')',\
+            'description': '',\
+            'args': args_dict,\
+            'duration': 0,\
+            'alert': alert,\
+            'message': None,\
+            'error': None,\
+            'status': 'PASS'}
+
 def test_make_bucket(client):
+    start_time = time.time()
+
+    log_output = initialize_log_output(client.make_bucket)
+    log_output['description'] = 'Tests make_bucket api'
+    # Get a unique bucket_name
+    log_output['args']['bucket_name'] = bucket_name = uuid.uuid4().__str__()
+
     is_s3 = client._endpoint_url.startswith("s3.amazonaws")
-    # Get unique bucket_name, object_name.
-    bucket_name = uuid.uuid4().__str__()
-    object_name = uuid.uuid4().__str__()
-    client.make_bucket(bucket_name)
-    # Check if bucket was created properly.
-    client.bucket_exists(bucket_name)
-    # Remove buckets
-    client.remove_bucket(bucket_name)
-    ## Check if return codes a valid from server.
+    try:
+        # Create a bucket
+        client.make_bucket(bucket_name)
+        # Check if bucket was created properly
+        client.bucket_exists(bucket_name)
+        # Remove bucket
+        client.remove_bucket(bucket_name)
+    except Exception as err:
+        log_output['message'] = err
+        log_output['error'] = traceback.format_exc()
+        log_output['status'] = 'FAIL'
+        log_output['duration'] = duration = round(time.time() - start_time, 2)
+        print(log_output)
```

Comment:
`print(dict)` doesn't output json format. you would need to use `json.dumps(dict)`

Reply:
Done.